packages/angular/projects/ui-angular/package.json

```diff
@@ -18,15 +18,29 @@
     "aws-amplify": "3.x.x || 4.x.x"
   },
   "dependencies": {
-    "@aws-amplify/ui-components": "^1.7.0",
+    "@angular-builders/jest": "^14.0.0",
     "@aws-amplify/ui": "3.12.4",
+    "@aws-amplify/ui-components": "^1.7.0",
     "@stencil/core": "2.8.0",
+    "@testing-library/angular": "^12.1.0",
     "nanoid": "3.1.31",
     "qrcode": "1.5.0",
     "tslib": "2.3.1",
     "xstate": "^4.30.6"
   },
   "publishConfig": {
     "directory": "../../dist/ui-angular"
+  },
+  "jest": {
+    "preset": "jest-preset-angular",
+    "setupFilesAfterEnv": [
+      "<rootDir>/src/setup.jest.ts"
+    ]
+  },
+  "devDependencies": {
+    "@types/jest": "^28.1.6",
+    "jest": "^28.1.3",
+    "jest-environment-jsdom": "^28.1.3",
+    "jest-preset-angular": "^12.2.0"
```

Comment:
Same as above -- this should be a devDependency at packages/angular/package.json if possible.